development-defaults.ini

```diff
@@ -169,3 +169,13 @@ group  = "Taking an unassigned Group badge (group leader must be present)"
 security = "The Dorsai Irregulars"
 
 [dept_head_checklist]
+
+
+[secret]
+test_db_file = "/tmp/uber.db"
+sqlalchemy_url = "sqlite+pysqlite:///%(test_db_file)s"
+
+# SQLite doesn't support these options, so they're set to -1 to indicate
+# that they should be ignored.
+sqlalchemy_pool_size = -1
+sqlalchemy_max_overflow = -1
```

Comment:
By setting these options here, you've put them in the ``[secret]`` section, which is probably not going to do the correct thing.  They're defined in our ``configspec.ini`` at the top level, so you should put these in the top section of ``development-defaults.ini`` rather than underneath any sub-section such as ``[secret]``.

Reply:
Actually I've just noticed a larger issue with these two options, which is that if you put them here, it's going to override them in production!  Normally this wouldn't be the case, because our puppet manifests populate the ``development.ini`` file which takes precedence over ``development-defaults.ini``.

However, a quick glance at ``development.ini`` in production shows that the ``sqlalchemy_pool_size`` and ``sqlalchemy_max_overflow`` options are never actually being set by puppet, so we can't set them here in this way.